inputfiles/addedTypes.jsonc

```
@@ -1195,6 +1195,63 @@
                         }
                     }
                 }
+            },
+            "UnderlyingByteSource": {
+                "name": "UnderlyingByteSource",
+                "members": {
+                    "member": {
+                        "type": {
+                            "name": "type",
+                            "required": true,
+                            "overrideType": "\"bytes\""
+                        },
+                        "autoAllocateChunkSize": {
+                            "name": "autoAllocateChunkSize",
+                            "overrideType": "number"
```

```suggestion
                            "type": "unsigned long long"
```

Kinda weird but I'd prefer `type` here whenever it works.